File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        list1 = [0] * 26
        list2 = [0] * 26
        for ch in s:
            list1[ord(ch) - ord('a')] += 1
        for ch in t:
            list2[ord(ch) - ord('a')] += 1
        
        for i in range(0, 26):
            if list1[i] != list2[i]:
                return False
        return True
            

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int list1[] = new int[26];
        int list2[] = new int[26];
        for(char ch : s.toCharArray()){
            list1[ch - 'a']++;
        }
        for(char ch : t.toCharArray()){
            list2[ch - 'a']++;
        }
        for(int i=0;i<26;i++){
            if(list1[i] != list2[i]){
                return false;
            }
        }
        return true;
    }
}